components/raftstore/src/store/worker/pd.rs

```diff
@@ -943,26 +964,36 @@ where
             let query_stats = region_peer
                 .query_stats
                 .sub_query_stats(&region_peer.last_store_report_query_stats);
+            let written_bytes =
+                region_peer.written_bytes - region_peer.last_store_report_written_bytes;
+            let written_keys =
+                region_peer.written_keys - region_peer.last_store_report_written_keys;
             region_peer.last_store_report_read_bytes = region_peer.read_bytes;
             region_peer.last_store_report_read_keys = region_peer.read_keys;
+            region_peer.last_store_report_written_bytes = region_peer.written_bytes;
+            region_peer.last_store_report_written_keys = region_peer.written_keys;
             region_peer
                 .last_store_report_query_stats
                 .fill_query_stats(&region_peer.query_stats);
             if read_bytes < hotspot_byte_report_threshold()
                 && read_keys < hotspot_key_report_threshold()
+                && written_bytes < hotspot_byte_report_threshold()
```

Comment:
`write` should have a different threshold. See https://github.com/tikv/pd/blob/ea57c6d6a0c9ad081aded76545f3d97bc465635f/server/statistics/hot_peer_cache.go#L46-L53